.changeset/seven-dots-serve.md

```diff
@@ -0,0 +1,34 @@
+---
+'@backstage/plugin-catalog': minor
```

Comment:
Is this `minor`? My scan of the code makes me think it's just `patch`, I don't see any breaking changes. Feel free to correct me on this one, good chance I'm missing something 🤔

Reply:
Based on [this in the Contributing](https://github.com/backstage/backstage/blob/d7a1adf48bc45634a95aff17c39621c3c586c1c8/CONTRIBUTING.md?plain=1#L181), I went with minor.

> Select impact of the change you're introducing. If the package you are changing is at version 0.x, use minor for breaking changes and patch otherwise. If the package is at 1.0.0 or higher, use major for breaking changes, minor for backwards compatible API changes, and patch otherwise. See the [Semantic Versioning specification](https://semver.org/#semantic-versioning-specification-semver) for more details.

Package is at 1.0.0 or higher and (to me) this is a backwards compatible API change

Reply:
Thanks for pointing that out. I knew there was different rules for 1.0 versions but should have looked